data/software.json

```diff
@@ -1647,6 +1647,15 @@
         "url": null
     },
     {
+        "categories": [
```

Comment:
This file is sorted alphabetically. Please move this entry to the appropriate position.